pkg/catalog/fake/fake.go

```diff
@@ -3,15 +3,13 @@ package fake
 import (
 	"time"
 
-	"github.com/openservicemesh/osm/pkg/compute"
-
 	"github.com/openservicemesh/osm/pkg/catalog"
 	tresorFake "github.com/openservicemesh/osm/pkg/certificate/providers/tresor/fake"
 	"github.com/openservicemesh/osm/pkg/messaging"
 )
 
-// NewFakeMeshCatalog creates a new struct implementing catalog.MeshCataloger interface used for testing.
-func NewFakeMeshCatalog(provider compute.Interface) *catalog.MeshCatalog {
+// NewFakeMeshCatalog creates a new catalog.MeshCatalog struct used for testing.
+func NewFakeMeshCatalog(provider catalog.Interface) *catalog.MeshCatalog {
```

Comment:
let's actually delete this whole file. Should be simple to remove all instances of it

Reply:
just curious on the reasoning behind deleting this? It's used in ~7 different tests so we'd just end up repeating this in those

Reply:
all it does is take in a compute interface, which is all we need to provide the real thing. so we can replace all instances with calls to catalog.NewMeshCatalog instead of this